bot/constants.py

```diff
@@ -10,11 +10,16 @@
 
 # Set BINANCE_NODE_IP depending on mode (if None, no Binance jobs are not executed)
 if DEBUG:
-    BINANCE_NODE_IP = 'localhost'
-elif 'BINANCE_NODE_IP' in os.environ and os.environ['BINANCE_NODE_IP']:
-    BINANCE_NODE_IP = os.environ['BINANCE_NODE_IP']
+    BINANCE_NODE_IPS = [binance_ip for binance_ip in
+                  "localhost,0.0.0.0".split(",")] \
+        if 'BINANCE_NODE_IPS' in os.environ and os.environ['BINANCE_NODE_IPS'] != "" \
+        else []
```

Comment:
```suggestion
    BINANCE_NODE_IPS = ['localhost', '0.0.0.0'] \
        if os.environ.get('BINANCE_NODE_IPS') \
        else []
```
Same simplification can be made below 
Is there any reason why we set `['localhost', '0.0.0.0']` instead of `['localhost']`?

Reply:
As I had issues with string splitting multiple IPs, I opted for a Debug version that resembled the production version the most.

You're right, let's use the array version which is cleaner.
I'd still go with 2 IPs to ensure proper functionality with mutliple IPs.